muesli/web/templates/tutorial/view.pt

```diff
@@ -6,91 +6,119 @@
+      <li><a tal:attributes='href request.route_path("lecture_email_tutors", lecture_id=tutorial.lecture.id)'>E-Mail an alle Übungsleiter und Assistenten schreiben</a></li>
+      <li tal:condition="request.permissionInfo.has_permission('edit') and len(tutorials) &lt; 2">
+        <a tal:attributes="href request.route_path('tutorial_edit', tutorial_id=tutorial.id)">${names['tutorial']} bearbeiten</a></li>
+      <li tal:condition="request.userInfo.is_tutor_of_tutorials(tutorials)">
+         <a tal:attributes="href request.route_path('tutorial_resign_as_tutor', tutorial_ids=tutorial_ids)">Als ${names['tutor']} zurücktreten</a></li>
+    </ul>
+  </div>
 </div>
 
-<form method="get" tal:attributes="action request.route_path('exam_correlation')">
-  <select name="source1">
-    <option tal:repeat="exam tutorial.lecture.exams" tal:attributes="value 'exam_%s' % exam.id">
-      ${exam.name}
-    </option>
-    <option tal:attributes="value 'lecture_%s' % tutorial.lecture.id">
-      Summe Übungszettel
-    </option>
-  </select>
-  <select name="source2">
-    <option tal:repeat="exam tutorial.lecture.exams" tal:attributes="value 'exam_%s' % exam.id">
-      ${exam.name}
-    </option>
-    <option tal:attributes="value 'lecture_%s' % tutorial.lecture.id">
-      Summe Übungszettel
-    </option>
-  </select>
-  <input type="submit" value="Korrelation"/>
-</form>
-
-<h3>Teilnehmer</h3>
-
-<table class="colored">
-
-<thead>
-  <tr>
-    <th>Name (E-Mail)</th>
-    <th>Studiengang</th>
-    <th tal:condition="request.permissionInfo.has_permission('edit')">Verschieben</th>
-    <th>Entfernen</th>
-  </tr>
-</thead>
-
-<tbody>
-  <tr tal:repeat="student students">
-    <td><a tal:attributes="href 'mailto:'+student.email">${student.name()}</a></td>
-    <td>${student.subject}</td>
-    <td tal:condition="request.permissionInfo.has_permission('edit')">
-      <metal:main metal:use-macro="templates('Fragments/HTML/move_student.pt').macros['movestudentmacro']" />
-    </td>
-    <td>
-      <form method="post" tal:attributes="action request.route_path('tutorial_remove_student', tutorial_ids=tutorial_ids, student_id=student.id)">
-        <input type="submit" value="Entfernen"/>
-      </form>
-    </td>
-  </tr>
-
-</tbody>
+<div class="card mt-2">
+  <div class="card-body">
+    <h3 class="card-title">Testate</h3>
+
+    <div tal:repeat="category categories" tal:omit-tag="">
+      <h4 tal:condition="exams[category['id']]">${category['name']}</h4>
+
+      <ul>
+        <li tal:repeat="exam exams[category['id']]">
+          <a tal:attributes="href request.route_path('exam_enter_points', exam_id=exam.id, tutorial_ids=tutorial_ids)">${exam.name}</a>
+          <a tal:attributes="href request.route_path('exam_enter_points_single', exam_id=exam.id, tutorial_ids=tutorial_ids)">(Interaktiv)</a>
+        </li>
+      </ul>
```

Comment:
Redesign this as well. Also rename „Interaktiv“ to „Punkte eintragen“. Let's advertise exam_enter_points as an overview of entered points, that happens to have an edit functionality

Reply:
The instructions for the redesign are a bit inconcrete here :P What exactly would you want to change?